app/browser/reducers/windowsReducer.js

```diff
@@ -35,6 +35,8 @@ const appDispatcher = require('../../../js/dispatcher/appDispatcher')
 const isDarwin = platformUtil.isDarwin()
 const isWindows = platformUtil.isWindows()
 
+const TIMEOUT_WINDOW_SHOW_MS = 5000
```

Comment:
I think this is fine here, but for future reference you may consider `js/constants/config.js` 😄

Reply:
moved to _config.js_